src/Features/Core/Portable/EditAndContinue/DebuggingSession.cs

```diff
@@ -745,7 +758,7 @@ public async ValueTask<ImmutableArray<ActiveStatementSpan>> GetAdjustedActiveSta
         {
             try
             {
-                if (_isDisposed || !EditSession.InBreakState || !mappedDocument.State.SupportsEditAndContinue())
+                if (_isDisposed || !EditSession.InBreakState || !mappedDocument.State.SupportsEditAndContinue() || !mappedDocument.Project.SupportsEditAndContinue())
```

Comment:
Is there a known scenario where the document supports enc but project doesn't? or is this just in case?

Reply:
It's "just in case".